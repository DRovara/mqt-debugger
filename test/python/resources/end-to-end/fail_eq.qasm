qreg q[2];

x q[0];
h q[1];

assert-eq q[0], q[1] { 0, 0.707, 0, 0.707}      // should fail
assert-eq 1.0, q[0], q[1] { 0, 0.707, 0, 0.707} // should fail
assert-eq 0.9, q[0], q[1] { 0, 0.707, 0, 0.707} // should pass

assert-eq q[1], q[0] { 0, 0.707, 0, 0.707}      // should fail
assert-eq 1.0, q[1], q[0] { 0, 0.707, 0, 0.707} // should fail
assert-eq 0.9, q[1], q[0] { 0, 0.707, 0, 0.707} // should fail

assert-eq q[1], q[0] { 0, 0, 0.707, 0.707}   // should fail
assert-eq 1.0, q[1], q[0] { 0, 0, 0.707, 0.707} // should fail
assert-eq 0.9, q[1], q[0] { 0, 0, 0.707, 0.707} // should pass
