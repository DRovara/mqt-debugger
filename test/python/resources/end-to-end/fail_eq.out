0, 0.707, 0, 0.707
---

---
7
